src/site/_data/paths/learn-web-vitals.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "learn-web-vitals",
+    "cover": "/images/collections/web-vitals.svg",
+    "title": "Web Vitals",
+    "description": "Key signals of quality user experiences.",
+    "overview": "Web Vitals define the performance metrics and measurements that indicate a delightful user experience.",
+    "topics": [
+        {
+            "title": "Overview",
+            "pathItems": [
+                "vitals"
+            ]
+        },
+        {
+            "title": "Web Vitals Metrics",
+            "pathItems": [
+                "lcp",
+                "fid",
+                "cls",
+                "defining-core-web-vitals-thresholds"
+            ]
+        },
+        {
+            "title": "Measuring Web Vitals",
```

Comment:
Per the [style guide](https://web.dev/handbook/grammar/#titles-and-headings), I think this needs to be "Measure Web Vitals"